client/main.js

```diff
@@ -0,0 +1,22 @@
+
+const getSentenceButton = document.getElementById("get-sentence-btn")
+const sentenceContainer = document.getElementById("sentence-container")
+
+const serverAddress = "localhost"
+const serverPort = 3000
+const getSentenceRoute = "greateSentence"
+
+const getSentenceURI = `http://${serverAddress}:${serverPort}/${getSentenceRoute}`
+
+function getSentenceFromServer() {
+    window.fetch(new Request(getSentenceURI))
```

Comment:
No need for creating a request. Simply giving fetch the uri will work too `fetch(getSentenceURI)`